src/mongo/integration/standalone/dbclient_writer_test.cpp

```diff
@@ -52,9 +52,11 @@ template <typename T>
 class DBClientWriterTest : public StandaloneTest {
 public:
     DBClientWriterTest() {
-        c.connect(server().uri());
```

Comment:
MongoResource::uri() has some parsing logic that basically gives you hostname:port